sdks/Cordova.md

```diff
@@ -0,0 +1,136 @@
+
+## Advantages of the solution 2
+
+- Ability to collect and link 3 different platfroms as part of the single repository
+- Code still can be reused thanks to keycloak, push, sync js implementations
+- No need for specific plugins for React and Cordova (platform specific wrappers can be provided) with single code base
+- No need to adjust native code for the plugins
+- No challenges related with cross js/native result access.
+
+See investigation details bellow for more details
+
+## Unanswered questions
+
+Questions that needs to be answered before proposal is merged.
+
+### Collecting all plugins into single repository
+
+Since some of the plugins like push are already in separate repository this can be seen as additional effort. 
+It may also complicate distribution as community preffers to have 1 plugin per repository and cordova cmd is optimized.
+I do not have strong opinions about this one but it feels like leaving things separate and having some aggregator repository with example application that links to all plugins and have parser code will be the best way.
+
+## Architecture for solution 1
+
+### Config parsing 
+
+Cordova config plugin not needed due to trivial implementation that can be done in pure JavaScript.
+Networking/Logging should be done on JavaScript side as this is common aproach for cordova.
+Due to trivial implementation no need for core module like in Android and IOS.
+
+### Individual SDK's
+
+Using existing cordova plugins or creating new cordova plugins for services.
+Following example is using auth module, but behaviour should be the same for the most of the plugins.
+Example:
+
+```
+// FIXME We need option without using browserify/webpack
+var config = require('mobile-config.json')
+var ConfigLib = require('aerogear-services')
+
+var keycloakConfig = ConfigLib(config).getKeyCloakConfig()
+
+cordova.auth.init(keycloak)
+
+// Using 
+cordova.auth.login()
+// Allow to retrieve token in javascript.
+cordova.auth.getToken()
+```
+
+This code relects how cordova based solutions work in general.
+
+## Investigation process
+
+All investigations and different variations of architecture will be available in following repository:
+https://github.com/wtrocki/CordovaSandbox
+
+### POC for Core
+
+Two options were investigated
+1. Wrapping core into plugin. 
+1. Writing pure JS helper.
+
+Wrapping core into plugin was proven difficult and innefective:
+- JavaScript implementation is trivial
+- Droping configuration into native projects may be bad user experience for cordova developers.
+- Core android IOS providing helpers that are available on cordova/javascript
```

Comment:
I don't get this point in the context of `Wrapping core into plugin was proven difficult and innefective`